test-cases/features/add-new-dc.yaml

```diff
@@ -0,0 +1,9 @@
+n_db_nodes: 3 0  # make n_db_nodes configured as multi-dc with last dc set to 0 (so later easily new node can be added)
```

Comment:
you could start this test with a multiDC cluster (like `n_db_nodes: 3 3`) and then add an extra DC

Reply:
that was intended - for dev purposes it was only 1 starting DC. Fixed.